src/test/java/test/sanitycheck/SuiteListner.java

```diff
@@ -0,0 +1,28 @@
+package test.sanitycheck;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.testng.ISuite;
+import org.testng.ISuiteListener;
+import org.testng.xml.XmlSuite;
+
+public class SuiteListner implements ISuiteListener {
+
+	List<XmlSuite> allSuite = new ArrayList<XmlSuite>();
```

Comment:
Private should be the right modifier. In Java its public by default. I think final is not necessary because we wont be changing the reference of the array list to any new array list.